### DISCUSSION TITLE
Other variants

### DISCUSSION THREAD

**Discussion by LeonKercmar** [OFFLINE-GmbH/oc-mall-plugin#1051] [Q&A]
Hey,

Is there any documentation on how to implement an other variants option like the one in the demo theme?

![image](https://github.com/OFFLINE-GmbH/oc-mall-plugin/assets/105213431/c771c9c9-f1e9-49fa-a956-a742a7850f2e)

Any help would be much appreciated.

**Comment by tobias-kuendig** [comment]
@LeonKercmar, did you see https://offline-gmbh.github.io/oc-mall-plugin/digging-deeper/products.html#examples. What exactly are you struggling with?

**Comment by LeonKercmar** [comment]
I'm struggling with implementing a dropdown with all the variants.
